server/middleware/authorisationMiddleware.ts

```diff
@@ -13,6 +14,11 @@ export default function authorisationMiddleware(authorisedRoles: string[] = []):
         return res.redirect('/authError')
       }
 
+      res.locals.user = {
+        roles,
+        isReceptionUser: roles.includes(Role.RECEPTION_USER),
```

Comment:
Kind of just plopped this bit of code in this middleware as it's relevant to authorisation and I didn't think it was big enough to create it's own middleware. Maybe if it becomes bigger (more complex access control based on roles) - we can separate into a **buildApplicationRoles** middleware or something along those lines